spec/javascripts/views/story_view_spec.js

```diff
@@ -665,4 +686,98 @@ describe('StoryView', function() {
       });
     });
   });
+
+  describe('attachmentStart', function() {
+    var $storyControls;
+
+    beforeEach(function() {
+      this.view.attachmentStart();
+    });
+
+    describe('in the control buttons', function() {
+      beforeEach(function() {
+        $storyControls = this.view.$el.find('.story-controls');
+      });
+
+      it('disable submit', function() {
+        var submit = $storyControls.find('.submit');
+        expect(submit.disabled).toBeTruthy;
+      });
+
+      it('disable cancel', function() {
+        var cancel = $storyControls.find('.cancel');
+        expect(cancel.disabled).toBeTruthy;
+      });
+
+      it('disable destroy', function() {
+        var destroy = $storyControls.find('.destroy');
+        expect(destroy.disabled).toBeTruthy;
+      });
+    });
+
+    describe('while the attchment is happening', function(){
+      it('update the uploadInProgress variable', function() {
+        expect(this.view.uploadInProgress).toBeTruthy;
+      });
+    });
+  });
+
+  describe('render', function() {
+    var $storyControls;
+
+    beforeEach(function() {
+      this.view.render();
+    });
+
+    describe('it render a enabled', function() {
+      beforeEach(function() {
+        $storyControls = this.view.$el.find('.story-controls');
+      });
+
+      it('submit button', function() {
+        var submit = $storyControls.find('.submit');
+
+        expect(submit.disabled).toBeFalsy;
+      });
+
+      it('cancel button', function() {
+        var cancel = $storyControls.find('.cancel');
+
+        expect(cancel.disabled).toBeFalsy;
+      });
+
+      it('destroy button', function() {
+        var destroy = $storyControls.find('.submit');
+
+        expect(destroy.disabled).toBeFalsy;
+      });
+    });
+
+    describe('it render a disabled', function() {
```

Comment:
I think that you can remove this block, its duplicate this one https://github.com/Codeminer42/cm42-central/pull/79/files#diff-a7b87a8a0623a4cc72cade623a107eccR697